### PR TITLE
docs: first clone then install reqs

### PR DESCRIPTION
**Pull request by daonb**
## Status

Ready for review

## Description of Changes

Getting started guide now first guides me through cloning the repo and then installing the requirments.

### If you made changes to documentation:

- [x] Doc linting passed locally

### CODE REVIEW

File: docs/development/getting_started.rst [hunk 1]
```diff
@@ -150,12 +146,29 @@ different version, the path to ``virtualenvwrapper.sh`` will differ. Running
...
+Ensure your virtualenv is activated and install the packages.
 
 .. code:: sh
 
-   git clone https://github.com/freedomofpress/securedrop.git
+    pip install -r securedrop/requirements/develop-requirements.txt
+    pip install -r testinfra/requirements.txt
+
+You will need to run this everytime new packages are added so if you get
+an ``ImportError`` run theose two lines again.
```
Comment (dachary) [inline docs/development/getting_started.rst]: The warning above could be:
<pre>
.. note:: You will need to run this everytime new packages are added.
</pre>
I think the 
<pre>
so if you get an ``ImportError`` run theose two lines again.
</pre>
can be left out.

**Comment by dachary** [comment]
> Doc linting passed locally

You can do this with `make docs-lint` . There are more details [in the documentation](https://docs.securedrop.org/en/stable/development/documentation_guidelines.html#testing-documentation-changes)

**Comment by daonb** [comment]
@dachary I've changed it based on your Input and squashed the commit.

**Comment by dachary** [comment]
Thanks for catching this and fixing it 💯

**Comment by dachary** [comment]
For the record I verified this looks good visually.

**Pull request merged**